config/initializers/rack_attack.rb

```diff
@@ -27,6 +30,34 @@ class Rack::Attack
     end
   end
 
+  throttle("get requests to Conversations API with token", limit: 10_000, period: 1.minute) do |request|
```

Comment:
```suggestion
  throttle("read requests to Conversations API with token", limit: 10_000, period: 1.minute) do |request|
```

tiny nit - just matches your terminology in the code

Reply:
ah great catch. Thanks!